tests/pairlist/test_pairlist.py

```diff
@@ -219,12 +243,16 @@ def test_VolumePairList_whitelist_gen(mocker, whitelist_conf, shitcoinmarkets, t
     freqtrade.pairlists.refresh_pairlist()
     whitelist = freqtrade.pairlists.whitelist
 
-    assert whitelist == whitelist_result
+    if type(whitelist_result) is list:
```

Comment:
Maybe add a comment here that this is needed to test shuffleFilter without seed ? 

I've had a different comment until i understood how this test works now.

Also - there is one thing i'd assert in addition here (this comes from my initial question in this line - if we have a filter/pairlist which cannot return a list ...).

```suggestion
    assert isinstance(whitelist_result, list)
    
    # Verify length of pairlist matches (used for ShuffleFilter without seed)
    if type(whitelist_result) is list:
```

Reply:
do you mean `assert isinstance(whitelist, list)` here? whitelist_result can now be int to test ShuffleFilter (and probably other filters in the future which could also return pairlist in diff. order)

Reply:
but that's what it says above :D 

(sorry - yes i misstyped before ...)

Reply:
Sorry - test-failure is most likely my bad - (as said before - i don't REALLY like the suggestion feature - as it makes such errors easy).